src/foam/comics/v2/DAOControllerConfig.js

```diff
@@ -51,6 +51,21 @@ foam.CLASS({
         return dao;
       }
     },
+    {
+      class: 'foam.dao.DAOProperty',
+      name: 'unfilteredDAO',
+      hidden: true,
+      expression: function(dao) {
+        var result = dao;
+        while ( dao && foam.dao.ProxyDAO.isInstance(dao) ) {
+          if ( foam.dao.FilteredDAO.isInstance(dao) ) {
+            result = dao.delegate;
```

Comment:
Simpler to just return dao.delegate?

Reply:
To not assume there is only one filtered dao in the chain, the real unfiltered dao is the delegate of the last filtered dao.

Reply:
Fixed in https://github.com/foam-framework/foam2/pull/4195/commits/a8d74914044caa6d22c810e3a63ff1bded612689.